### ISSUE TITLE
Quick tutorial for newbs?

### ISSUE THREAD

**Issue by nathanieltagg** [hustvl/SparseInst#94]
Hi, we're interested in evaluating this model, and we're trying to run it over our custom dataset with our custom object, so we have only one class, and usually just have a handful of instances in an image.
We've tried loading the R-50 model and setting
cfg.MODEL.SPARSE_INST.DECODER.NUM_MASKS = 10
cfg.MODEL.SPARSE_INST.DECODER.NUM_CLASSES = 1
This generates errors when loading the pre-trained model.  It does actually train, but objects have very bad boundaries.  The loss function does not seem to be converging after the first few iterations, and bounces up and down further than we'd expect.

Would it be possible to have a quick example showing how to add an object class, or any pointers on things to try?
Thanks!

**Comment by wondervictor** [comment]
Hi @nathanieltagg, thanks for your interest in SparseInst!
1. How about setting `NUM_MASKS=100`, which keeps the pre-trained weights of the IAM?
2. Could you tell me which configuration is adopted? 
3. And could you provide some brief details about your custom dataset, such as the amount, resolution, or other noteworthy features?
4. How about the annotation quality of the segmentation?

**Comment by nathanieltagg** [comment]
@wondervictor Thanks for the response!   I turned down masks in the hopes that it would reduce parameter space, but I'll try the other way too.  
So far we've tried the R-50 and the R-50-giam models, without much change. We're trying to segment the face of loading pallets. We've got about 8k labelled images at 1080p, most with a single instance but a few with mutliple instances.   The annotation quality is quite good; we've trained an RCNN detectron model with great success, but we're after more speed and better segmentation with obscruring obstacles.

Any suggestions would be welcome!  Thanks!

![test](https://user-images.githubusercontent.com/2135685/199729777-6d4323b0-3a82-48c4-8177-13790ee482aa.jpg)
